mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewclient/CreateNewClientFragment.java

```diff
@@ -106,7 +128,9 @@
     CreateNewClientPresenter createNewClientPresenter;
 
     View rootView;
+    private boolean createClientWithImage = false;
```

Comment:
Provide some better name which clarifies its use.  It seems like a flag to save the state. Alternatively, (though less preferred) add a comment here about the variable usage so that future developers may get idea about why it was used.